lib/resources/assets.tsx

```diff
@@ -529,7 +544,9 @@ const backendSearch = async (
         versionIndex = index;
       }
     });
-    store.dispatch(updateCourseCache(courses.data.data));
+    const cache: SISRetrievedCourse[] = [];
+    cache.push(retrieved);
+    store.dispatch(updateCourseCache(cache));
```

Comment:
backendSearch is a function that searches by course number. 
this means retrieved course is of interest. 
cache it.